src/verify.py

```diff
@@ -0,0 +1,13 @@
+array = [2,4,6,8,10,12,14,16,18,20]
```

Comment:
sarebbe meglio passare l'array direttamente come parametro della funzione verify, perché vogliamo che la funzioni verifichi se un'elemento si trova dentro un dato array, così facendo invece verify controllerebbe solo se l'elemento si trova in questo particolare array che tu hai deciso :)

Reply:
non capisco, la consegna stesso indica di verificare la presenza dell'elemento un un array pre-defined...

Reply:
hai ragione! Errore mio 😅